dcpy/builds/load.py

```diff
@@ -23,10 +23,6 @@ def import_dataset(
     pg_client: postgres.PostgresClient,
 ):
     """Import a recipe to local data library folder and build engine."""
-    ds_table_name = ds.import_as or ds.name
-    logger.info(
-        f"Importing {ds.name} into {pg_client.database}.{pg_client.schema}.{ds_table_name}"
```

Comment:
This also gets logged in the called recipes.import_dataset, meaning this was printed twice each time before